Libraries/Components/Touchable/TouchableHighlight.js

```diff
@@ -33,6 +33,11 @@ type Event = Object;
 var DEFAULT_PROPS = {
   activeOpacity: 0.85,
   underlayColor: 'black',
+  tvParallaxDisable: false,
+  tvParallaxShiftDistanceX: 2.0,
+  tvParallaxShiftDistanceY: 2.0,
+  tvParallaxTiltAngle: 0.05,
+  tvParallaxMagnification: 1.0
```

Comment:
Is there anyway we can move these to the native sides and only apply defaults when you're running on TVOS?